test/nools/lib.tasks.spec.js

```diff
@@ -104,6 +140,42 @@ describe('nools lib', function() {
         ]);
       });
 
+      it('appliesToType filters by form', () => {
+        // given
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [ aReportBasedTask() ],
+        };
+        config.tasks[0].appliesToType = ['dne'];
+
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        expect(emitted).to.have.length(1);
+      });
+
+      it('appliesToType is not required', () => {
+        // given
+        const config = {
+          c: personWithReports(aReport()),
+          targets: [],
+          tasks: [ aReportBasedTask() ],
+        };
+        delete config.tasks[0].appliesToType;
+
+        // when
+        const { emitted } = runNoolsLib(config);
+
+        // then
+        console.log(JSON.stringify(emitted, null, 2));
```

Comment:
Logging again.
It's weird, it looks like running `npm test` on Node > 8 will only log some form conversion  warnings and mocha reporting is completely silenced. Both on Travis and locally.